holoviews/core/decollate.py

```diff
@@ -0,0 +1,255 @@
+import param
+from .operation import OperationCallable
+from .. import (
+    Layout, DynamicMap, Element, Callable, Overlay, GridSpace, NdOverlay, HoloMap
+)
+from . import ViewableTree
+from collections import namedtuple
+from ..streams import Stream, Derived
+
+from ..plotting.util import initialize_dynamic
+
+HoloViewsExpr = namedtuple("HoloviewsExpr", ["fn", "args", "kwargs"])
```

Comment:
Given that everything here is HoloViews, why not just name it `Expr`? :-)

Reply:
sure!

Reply:
Done in https://github.com/holoviz/holoviews/pull/4551/commits/1e0aeb67d938e7342ef7bca3f948833f4c5cdd4e